tools/helper.psm1

```diff
@@ -17,8 +17,8 @@ $DotnetSDKVersionRequirements = @{
     }
 
     '6.0' = @{
-        MinimalPatch = '100'
-        DefaultPatch = '100'
+        MinimalPatch = '101'
```

Comment:
Should the MinimalPatch and DefaultPatch for 3.1 be updated to 416?

Reply:
Thank you for pointing this out. Yes, we should. Fixed.